app/views/users/users/new.html.erb

```diff
@@ -1,61 +1,110 @@
 <% provide(:title, "Nutzer neu") %>
 
-<div class="content-wrapper">
+<div class="content-wrapper user-new">
   <div class="content-box">
     <div class="box-header">Neuer Nutzer</div>
     <%= form_for(@user, url: user_index_path) do |f| %>
+        <table class="user-new-table">
+          <tr>
+            <td>
+              Name
+            </td>
+            <td>
+              <div class="login-field">
+                <%= f.text_field :first_name, placeholder: "Vorname" %>
```

Comment:
Würde sagen gleich Sprachen vereinheitlichen wenn man schon an den code arbeitet oder lieber doch in ein extra issu?

Sonst umbauen auf Sprachpakete in configuration